decode/Cargo.toml

```diff
@@ -10,6 +10,7 @@ workspace = true
 [dependencies]
 clap = { version = "4.5.46", features = ["derive"] }
 clap-stdin = "0.6.0"
+mls-rs = "0.50.1"
 proteus-wasm = { workspace = true, features = ["serde"] }
 base64 = { workspace = true }
-openmls = { workspace = true }
+#openmls = { workspace = true }
```

Comment:
If we're not going to use openmls in decode, we should just remove this line, whoever needs it later can have a look at git history.